addons/sale/sale_view.xml

```diff
@@ -434,11 +485,10 @@
             <field name="type">ir.actions.act_window</field>
             <field name="res_model">sale.order</field>
             <field name="view_type">form</field>
-            <field name="view_id" ref="view_quotation_tree"/>
+            <field name="view_id" ref="view_sales_order_filter"/>
```

Comment:
why this change? in view_id you changed tree view reference to search view. wrong. correct it. keep as it is.